### ISSUE TITLE
Checksum Mismatch

### ISSUE THREAD

**Issue by parkerduckworth** [StackExchange/wmi#66]
Hello!

While fetching a Golang lib that depends on this one, I encountered a warning that indicated a mismatch between the checksum downloaded, and that of the checksum server: 

```
$ go get github.com/StackExchange/wmi
github.com/StackExchange/wmi@v1.2.0: verifying module: checksum mismatch
        downloaded: h1:BfLCNdXnvwgy5RrRI3IyQ64ZItZngXHN+7PxU5RvKxA=
        sum.golang.org: h1:noJEYkMQVlFCEAc+2ma5YyRhlfjcWfZqk5sBRYozdyM=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.
```

I know this project is no longer maintained, but if this is a security issue, figured it was worth bringing it up anyway.  Thanks yall

**Comment by parkerduckworth** [comment]
Just noticed that this problem is indirectly referenced by issue #62, but searching for "checksum" or "mismatch" does not return any issue search results, so leaving this issue open for visilbity

**Comment by xuyang2** [comment]
found a workaround:
add this to your module's `go.mod`:
```

replace (
	github.com/StackExchange/wmi v1.2.0 => github.com/StackExchange/wmi v1.2.1-0.20210723131906-441642c16659
)
```

**Comment by parkerduckworth** [comment]
Closing, as this appears to have been resolved (ref: #62)
